pkg/controller/installation/core_controller.go

```diff
@@ -1392,9 +1404,14 @@ func calicoDirectoryExists() bool {
 	return true
 }
 
-func (r *ReconcileInstallation) SetDegraded(reason string, err error, log logr.Logger) {
-	log.Error(err, reason)
-	r.status.SetDegraded(reason, err.Error())
+//func (r *ReconcileInstallation) SetDegraded(reason string, err error, log logr.Logger) {
+//	log.Error(err, reason)
+//	r.status.SetDegraded(reason, err.Error())
+//}
+
+func (r *ReconcileInstallation) SetDegraded(reason operator.TigeraStatusReason, message string, err error, log logr.Logger) {
+	log.Error(err, string(reason)+message)
+	r.status.SetDegraded(string(reason), message+err.Error())
```

Comment:
I imagine this does not look pretty. You probably want to use `fmt.Sprintf(...)` to format this nicely.